ert/data/record/_record.py

```diff
@@ -81,26 +113,73 @@ def __init__(self, data: blob_record_data) -> None:
 
     @beartype
     def _validate_data(self, data: blob_record_data) -> blob_record_data:
+        """Validates whether the data are of `blob_record_data` type via
+        beartype lib.
```

Comment:
Is beartype relevant here, or should we rather write what the criterias given to beartype are?

Reply:
Yeah, agree. Beartype in itself does not provide anything. "Runtime type validation" does.

Reply:
It's been updated